CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Contributing Instructions
+
+### Setup
+
+```sh
+# install golang, then:
+go get -u github.com/goware/modvendor
+make vendor
```

Comment:
Maybe we can have this as a separate "When you update dependencies, run the following command to vendor all included files:" (since its technically not needed during setup, only when you're adding new modules)